docs/register_agreement.yml

```diff
@@ -0,0 +1,54 @@
+Register a ServiceAgreement into MetadataDB
```

Comment:
We need to rename the endpoints. We are no longer storing service agreements

Reply:
Also the integration tests were written assuming that the users had access to the service agreement so on the javascript sdk implementation that needs to be changed to support this instead

Reply:
> I think the index looks simpler and better now. One last consideration, do you think it would be useful to register in that index a publication date time or similar (`createdAt` as `datetime`)? That way that index can be used to create the secondary market wall where you can sort by date when the asset was made available in that market.

I think we should go with just storing the entire service agreement. It has a well defined schema and then the index would work for any agreement and the creation date is already there